pkgs/development/python-modules/APScheduler/default.nix

```diff
@@ -11,20 +11,19 @@
 , tornado
 , twisted
 , mock
-, trollius
 , gevent
 , six
 , pytz
 , tzlocal
 , funcsigs
-, futures
 , setuptools
-, isPy3k
+, isPy27
 }:
 
 buildPythonPackage rec {
   pname = "APScheduler";
   version = "3.7.0";
+  disabled = isPy27;
```

Comment:
The package also does not support any version below 3.7.

Reply:
Right. Fixed.